esphome/components/color_temperature/light.py

```diff
@@ -22,6 +25,12 @@
             cv.Required(CONF_BRIGHTNESS): cv.use_id(output.FloatOutput),
             cv.Required(CONF_COLD_WHITE_COLOR_TEMPERATURE): cv.color_temperature,
             cv.Required(CONF_WARM_WHITE_COLOR_TEMPERATURE): cv.color_temperature,
+            cv.Optional(CONF_OFF_COLOR_TEMPERATURE): cv.enum(
+                {
+                    CONF_COLD_WHITE: None,
+                    CONF_WARM_WHITE: None,
+                }
+            ),
```

Comment:
this should be `cv.one_of`
```suggestion
            cv.Optional(CONF_OFF_COLOR_TEMPERATURE): cv.one_of(CONF_COLD_WHITE, CONF_WARM_WHITE, lower=True),
```